app/partials/main-header.cjsx

```diff
@@ -70,19 +76,19 @@ module.exports = React.createClass
 
   links: ->
     <nav className="main-nav #{@mobileClass()}">
-      <Link to={"/projects"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.projects" /></Link>
-      <Link to={"/about"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.about" /></Link>
-      <Link to={"/talk"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.talk" /></Link>
-      <Link to={"/notifications"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.notifications" /></Link>
-      <Link to={"/collections"} className="main-nav-item" activeClassName="active"><Translate content="mainNav.collect" /></Link>
+      <Link to={"/projects"} className="main-nav-item" activeClassName="active" onClick={@logClick?.bind(this, 'mainNav.projects')}><Translate content="mainNav.projects" /></Link>
```

Comment:
these event type keys have camel case and dots, where a lot of the others have all lower case and dashes. Up to now all Geordi event types have been lower case and dashes. I think we should keep them consistent unless we have a good reason not to.